index.html

```diff
@@ -4805,8 +4805,62 @@ <h2>
 layers, and pad messages to standard lengths.
       </p>
     </section>
-  </section>
 
+    <section>
+      <h2>
+          Service Privacy
+      </h2>
+      <p>
+The ability for a controller to optionally state at least one service endpoint
+in the DID document increases their control and agency. Stating more than one
+endpoint in the DID document always adds privacy risk either due to correlation
+across the endpoint descriptions or because the services are not protected by
+an authorization mechanism, or both.
```

Comment:
Adding even one adds privacy risk. Perhaps this could be reworded like so:

```suggestion
in the DID document increases their control and agency. Each additional
endpoint in the DID document adds privacy risk either due to correlation
(e.g., across endpoint descriptions) or because the services are not
protected by an authorization mechanism, or both.
```

Reply:
I'm fine with this change, but this PR was intended to embody @agropper 's verbiage, so I don't feel like I can unilaterally update it. Adrian?